app/src/main/java/com/blockeq/stellarwallet/mvvm/effects/EffectsViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package com.blockeq.stellarwallet.mvvm.effects
+
+import android.app.Application
+import android.arch.lifecycle.AndroidViewModel
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import com.blockeq.stellarwallet.mvvm.effects.remote.RemoteRepository
+import org.stellar.sdk.responses.effects.EffectResponse
+import java.util.*
+
+
+class EffectsViewModel(application: Application) : AndroidViewModel(application) {
+
+    private val effectsRepository : EffectsRepository = EffectsRepository.getInstance(RemoteRepository.instance)
```

Comment:
`getInstance(instance)`... two singletons. It is not very nice, repository should be the only singleton the other should be a regular class.